mqtt_io/config/config.schema.yml

```diff
@@ -250,7 +250,24 @@ mqtt:
           type: boolean
           required: no
           default: false
-
+    reconnect-delay:
```

Comment:
The convention is to use underscores instead of hyphens in the config file.

Reply:
Fixed